rpm/0001-Disable-usage-of-LO_FLAG_DIRECT_IO.patch

```diff
@@ -0,0 +1,26 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Pekka Vuorela <pekka.vuorela@jolla.com>
+Date: Wed, 15 Jan 2025 14:58:57 +0200
+Subject: [PATCH] Disable usage of LO_FLAG_DIRECT_IO
+
+Introduced in linux kernel 4.10. This change can be omitted once
+we're on that or newer.
+---
+ src/plugins/loop.c | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/src/plugins/loop.c b/src/plugins/loop.c
+index 835be812..83ba9ab9 100644
+--- a/src/plugins/loop.c
++++ b/src/plugins/loop.c
+@@ -176,8 +176,8 @@ BDLoopInfo* bd_loop_info (const gchar *loop, GError **error) {
+     info->offset = li64.lo_offset;
+     if ((li64.lo_flags & LO_FLAGS_AUTOCLEAR) != 0)
+         info->autoclear = TRUE;
+-    if ((li64.lo_flags & LO_FLAGS_DIRECT_IO) != 0)
+-        info->direct_io = TRUE;
++//    if ((li64.lo_flags & LO_FLAGS_DIRECT_IO) != 0)
++//        info->direct_io = TRUE;
```

Comment:
Please the `LINUX_VERSION_CODE` `KERNEL_VERSION(a,b,c)` macro here to disable the feature conditionally. 

E.g. as in:
```suggestion
#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,10,0)
  if ((li64.lo_flags & LO_FLAGS_DIRECT_IO) != 0)
  info->direct_io = TRUE;
#endif 
```

Reply:
That's a good idea. Updated to such